make/golang-v1.mk

```diff
@@ -176,19 +139,14 @@ endef
 define golang-build
 @echo "BUILDING..."
 @if [ -z "$$CI" ]; then \
-	go build -o bin/$(2) $(1); \
+	go build $(GO_BUILD_FLAGS) -o bin/$(2) $(1); \
 else \
 	echo "-> Building CGO binary"; \
-	CGO_ENABLED=0 go build -installsuffix cgo -o bin/$(2) $(1); \
+	CGO_ENABLED=0 go build $(GO_BUILD_FLAGS) -installsuffix cgo -o bin/$(2) $(1); \
 fi;
 endef
 
 # golang-update-makefile downloads latest version of golang.mk
 golang-update-makefile:
-	@wget https://raw.githubusercontent.com/Clever/dev-handbook/master/make/golang.mk -O /tmp/golang.mk 2>/dev/null
-	@if ! grep -q $(GOLANG_MK_VERSION) /tmp/golang.mk; then cp /tmp/golang.mk golang.mk && echo "golang.mk updated"; else echo "golang.mk is up-to-date"; fi
-
-# golang-major-update-makefile downloads latest version of golang.mk to the next major version, 1.x.x
-golang-major-update-makefile:
```

Comment:
Makes sense that this will be just in the old version, with the idea that if we ever have a v2, we can then add a new target to the v1 Makefile accordingly.

Maybe name it `golang-major-update-makefile-v1` or something to make it clear that it's not necessarily jumping to the latest major version, just to that specific major version.